modules/backend/behaviors/RelationController.php

```diff
@@ -1329,12 +1398,32 @@ public function relationExtendManageWidget($widget, $field, $model)
      * Provides an opportunity to manipulate the pivot widget.
      * @param Backend\Classes\WidgetBase $widget
      * @param string $field
-     * @param October\Rain\Database\Model $model
+     * @param \October\Rain\Database\Model $model
      */
     public function relationExtendPivotWidget($widget, $field, $model)
     {
     }
 
+    /**
+     * Provides an opportunity to manipulate the manage filter widget.
+     * @param \Backend\Widgets\Filter $widget
+     * @param string $field
+     * @param \October\Rain\Database\Model $model
+     */
+    public function relationExtendManageFilterWidget($widget, $field, $model)
```

Comment:
@alxy I might suggest renaming this to `relationExtendManageFilterScopes` and changing the first attribute to `$filter` so it is consistent with the list widget filter extensions (https://octobercms.com/docs/backend/lists#extend-filter-scopes)
```suggestion
    public function relationExtendManageFilterScopes($filter, $field, $model)
```

Reply:
I made these callbacks consistent with the other hooks, e.g. `relationExtendManageWidget`.
I also have to say that the callback is indeed more generic and does not only allow for extending the filter scopes (although this might be the most common use case), but for modifying the entire Filter widget. This is why I think the name is ok.

Reply:
Technically, a dev also has the same scope of access with the `extendListFilterScopes` method as well, being that the first parameter there is also an instance of `\Backend\Widgets\Filter`. I figured that renaming this method might also add some separation from the similarly named `relationExtendManageWidget` method.

@w20k @LukeTowers do you guys have any thoughts on this?

Reply:
We could change only an attribute name and call it, something like, - `$filterWidget`. And that should be it 😄

`public function relationExtendManageFilterWidget($filterWidget, $field, $model)`

Reply:
Thinking about it, I think it's fine to keep the name as is. It's probably `extendListFilterScopes` that should be renamed :)